quickstart-templates/workload-identity-creates-workload-identities/README.md

```diff
@@ -0,0 +1,70 @@
+# Create Workload Identity Federation that can create other workload identity federations
```

Comment:
I wonder whether to fully round out the scenario, that we should provide another Bicep template - i.e an example template that is created by and checked-in by project developers, picked up by the CI/CD pipeline and deployed via the privileged Creator workload identity.:

_folder structure:_
workload-identity-creates-workload-identities
- README.md
- Creator-workload-identity
   - bicepconfig.json
   - main.bicep
   - main.bicepparam
- Repo-workload-identity-example
    - bicepconfig.json
   - main.bicep
   - main.bicepparam
     
Thoughts?